spec/rubocop/cop/metrics/method_length_spec.rb

```diff
@@ -4,7 +4,7 @@
   subject(:cop) { described_class.new(config) }
   let(:cop_config) { { 'Max' => 5, 'CountComments' => false } }
 
-  it 'rejects a method with more than 5 lines' do
+  it 'rejects a method with more than 5 lines, syntax #1' do
```

Comment:
I'd surround this with `context`. `syntax #1` and `syntax #2` is too vague.

Reply:
@bbatsov There was some duplication in the spec, so I took the liberty to refactor it a little bit. If you think it's too much, I can revert to the original version, and just add a context around the new example.